src/libraries/Microsoft.PowerFx.Json/FormulaValueJSON.cs

```diff
@@ -54,7 +54,12 @@ public static FormulaValue FromJson(JsonElement element, FormulaType formulaType
                     return FormulaValue.NewBlank(formulaType);
 
                 case JsonValueKind.Number:
-                    if ((skipTypeValidation && numberIsFloat) || formulaType is NumberType)
+                    if (formulaType is DateTimeType)
+                    {
+                        // unixtime
+                        return DecimalValue.New(element.GetInt64());
```

Comment:
>DecimalValue [](http://example.com/codeflow?start=31&length=12)

This has a type mismatch (FormulaType says we expect a DateTime, but we're returning a Decimal)  - will that be a problem?